.github/workflows/bsd.yaml

```diff
@@ -60,3 +60,113 @@ jobs:
             gmake -C regression/cbmc test-paths-lifo
             env PATH=$PATH:`pwd`/src/solvers gmake -C regression/cbmc test-cprover-smt2
             # gmake -C jbmc/regression test-parallel JOBS=3
+
+  # This job takes approximately X to Y minutes
+  openbsd:
+    runs-on: macos-12
```

Comment:
Is it possible to run these on linux machines rather than macOS ones?

I'm asking because I've noticed that macOS based runners are always slower (sometimes **much** slower) in acquiring a VM to run the action on.

Reply:
This is possible, but I was hoping that nested virtualisation (as available on macos, but not on Linux) would provide for better performance than QEMU. The performance, however, seems surprisingly poor, so perhaps going via QEMU isn't actually any slower. I'll experiment.